README.md

```diff
@@ -82,6 +82,17 @@ should be called to update the UI.
 This in turn will invoke the `build` method 
 which re-renders the widget.
 
+## Flutter application test
+
+As in all languages or platforms the secret to a successful application is to test it.
+Several tests should be done like the functionality test, the user experience test, which can help us correct any bug.</br>
+
+An application must have tested any function, any class or task needed to run correctly without errors.</br>
+
+In general, an application that wants to succeed must have performed several unit tests, these being the main ones since they test functions, classes alone.</br>
```

Comment:
```suggestion

```
This sentences does not add much that has not been said already. 
Functional tests are unit tests.
see: https://stackoverflow.com/questions/2741832/unit-tests-vs-functional-tests